libaleth-interpreter/VM.cpp

```diff
@@ -758,11 +760,10 @@ void VM::interpretCases()
             ON_OP();
             updateIOGas();
 
-            static u256 const hibit = u256(1) << 255;
-            static u256 const allbits =
-                u256("0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
+            static intx::uint256 const hibit = intx::uint256(1) << 255;
```

Comment:
This could be better `constexpr`, and also can you use custom literals here, like `1_u256`?